source/1_engine/rex_engine/src/filesystem/vfs.cpp

```diff
@@ -311,6 +286,42 @@ namespace rex
           });
     }
 
+    void init(rsl::string_view root)
+    {
+      if(rsl::optional<rsl::string_view> arg = cmdline::get_argument("Root"))
+      {
+        root = arg.value();
+      }
+
+      if(root.empty())
+      {
+        WIN_CALL(GetCurrentDirectoryA(g_root.max_size(), g_root.data()));
+        g_root.reset_null_termination_offset();
+      }
+      else if(is_abs(root))
+      {
+        g_root = root;
+      }
+      else
+      {
+        rsl::medium_stack_string cwd;
+        WIN_CALL(GetCurrentDirectoryA(cwd.max_size(), cwd.data()));
+        cwd.reset_null_termination_offset();
+
+        g_root = cwd;
+        g_root += "/";
+        g_root += root;
```

Comment:
Can't we use a more platform agnostic approach here?
path::combine?

Reply:
not part of this PR I'd say